src/Tribe/Rewrite.php

```diff
@@ -10,6 +10,20 @@
  * Permalinks magic Happens over here!
  */
 class Tribe__Events__Rewrite extends Tribe__Rewrite {
+
+	/**
+	 * Constant holding the transient key for delayed triggered flush from activation.
+	 *
+	 * If this value is updated make sure you look at the method in the main class of TEC.
+	 *
+	 * @see TEC::activate
+	 *
+	 * @since 5.0.0.1
+	 *
+	 * @var string
+	 */
+	const KEY_DELAYED_FLUSH_REWRITE_RULES = '_tribe_events_delayed_flush_rewrite_rules';
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 88 characters

<sub>hash: fa439bf56449795abcaeafd38d8b4274</sub>